src/gui/ConfigurationDialog.cpp

```diff
@@ -1692,3 +1694,19 @@ void ConfigurationDialog::setDitherFormat()
 	conf->setValue("video/dithering_mode", selectedFormat);
 	conf->sync();
 }
+
+void ConfigurationDialog::selectAppFont()
+{
+	QFont font=QGuiApplication::font();
+	int size=font.pixelSize();
```

Comment:
Unused variable

Reply:
Come on - this is work in progress. If you activate line 1706, it will be used. I'd like to get comments on features, not trivialities.